bvm/ballerina-runtime/src/main/resources/MessagesBundle_en_US.properties

```diff
@@ -160,3 +160,4 @@ tuple.index.out.of.range = tuple index out of range: index: {0}, size: {1}
 illegal.array.insertion = array of length {0} cannot be expanded into array of length {1} without filler values
 java.null.reference = handle refers to Java null
 illegal.tuple.insertion = tuple of length {0} cannot be expanded into tuple of length {1} without filler values
+incompatible.type.assignment = ''{0}'' cannot be assigned to type ''{1}''
```

Comment:
We do not need to put this in here except any difference to MessagesBundle.properties message.

Reply:
But most of the previously added messages also same in the both files. We are maintaining two message bundle properties for English and English(US) and they should reflect the native way of expressions of both languages. But US one is not being updated recently for some newly added error messages. Since it is there for a reason (might be for language selection), INS not adding to this is a bug.

Reply:
IMO we need this only if we need change different message to English(US) . If it is not there MessagesBundle.properties will used. Here we have same message in both files then no need.